ui/app/components/job-status/panel.js

```diff
@@ -0,0 +1,81 @@
+// @ts-check
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class JobStatusPanelComponent extends Component {
+  // Build note: allocTypes order matters! We will fill up to 100% of totalAllocs in this order.
+  allocTypes = [
+    'running',
+    'pending',
+    'failed',
+    // 'unknown',
+    // 'lost',
+    'queued',
+    'complete',
+    'remaining',
+  ].map((type) => {
```

Comment:
I fully expect this list to be in a state of change for the next few weeks/months until we nail down what we want to show and what we want to aggregate.